pyanaconda/modules/common/structures/storage.py

```diff
@@ -453,3 +453,61 @@ def get_root_device(self):
         :return: a device name or None
         """
         return self.mount_points.get("/")
+
+
+class PlatformMountPointData(DBusData):
+    """Platform required mount points data"""
+
+    def __init__(self):
+        self._value = ""
+        self._filesystem = ""
+        self._encrypted = False
+        self._lv = False
```

Comment:
My first thought was that these could have actually descriptive names. Something along these lines...
- value = mountpoint, path
- encrypted = can_be_encrypted, encryption_allowed
- lv = can_be_on_lv, allowed_on_lvm...

eg. "lv" feels to me like "this is lv", not "this can be on lv"...

I see that these match `PartSpec`, so it's clear where this came from. However, for a D-Bus consumer, they will see first the names, so it might be good to make them less cryptic?